src/main/java/org/folio/rest/impl/PurchaseOrderHelper.java

```diff
@@ -321,6 +331,18 @@ private int calculateTotalItemsQuantity(List<CompositePoLine> poLines) {
     return validateOrder(compPO);
   }
 
+  private CompletableFuture<CompositePurchaseOrder> createUnitAssignments(CompositePurchaseOrder comPO) {
+    CompletableFuture<CompositePurchaseOrder> future = new CompletableFuture<>();
+    CompletableFuture.allOf(comPO.getAcqUnitIds().stream().map(id -> acquisitionsUnitAssignmentsHelper
+      .createAcquisitionsUnitAssignment(new AcquisitionsUnitAssignment())).toArray(CompletableFuture[]::new))
```

Comment:
The acquisition Unit assignment will not be created for the record here, just a new Object is being sent with no values

Reply:
Done.